tests/10apidoc/30room-create.pl

```diff
@@ -426,7 +426,9 @@ =head2 matrix_create_room_synced
 
 The parameters are passed through to C<matrix_create_room>.
 
-The resultant future completes with the room_id.
+The resultant future completes with the room_id and a room_alias.
+If room_alias_name is present in %params, an alias will be built with the given 
+alias and user's server name. Othwerwise, it will return undef.
```

Comment:
Odd line wrapping (and a typo) here:

```suggestion
The resultant future completes with the room_id and a room_alias. If
room_alias_name is present in %params, an alias will be built with the given 
alias and user's server name. Otherwise, it will return undef.
```

Ideally the symbols in that text (`room_alias_name`, `%params`, `undef`) would be wrapped with a `C<...>` [formatting code](https://perldoc.perl.org/perlpod#Formatting-Codes), but it's not critical.

Reply:
Please can you update the example on line 422 as well?